go/obscuronode/host/host.go

```diff
@@ -73,6 +75,9 @@ type Node struct {
 
 	// Node nodeDB - stores the node public available data
 	nodeDB *DB
+
+	// is the node ready
```

Comment:
@otherview Can we clarify here - ready for what? Is it ready for P2P messages from Obscuro peers?

Reply:
Good point - let me add more info